projects/cds/src/merchandising/model/merchandising.products.ts

```diff
@@ -0,0 +1,6 @@
+import { Product } from '@spartacus/core';
```

Comment:
To keep model files consistent, can you please rename the model classes to end with `.model.ts`?

Reply:
Renamed